pkg/api/const.go

```diff
@@ -61,6 +61,8 @@ const (
 	Kubernetes162 OrchestratorVersion = "1.6.2"
 	// Kubernetes166 is the string constant for Kubernetes 1.6.6
 	Kubernetes166 OrchestratorVersion = "1.6.6"
+	// Kubernetes166 is the string constant for Kubernetes 1.6.6
+	Kubernetes170 OrchestratorVersion = "1.7.0"
 	// KubernetesLatest is the string constant for latest Kubernetes version
 	KubernetesLatest OrchestratorVersion = Kubernetes166
```

Comment:
We like to let a new version back a little while before making it default, possibly we should rename this.